content/docs/languages/go/quickstart.md

```diff
@@ -17,12 +18,12 @@ weight: 10
 
 - **Go plugin** for the protocol compiler:
 
-   1. Install the protocol compiler plugin for Go (`protoc-gen-go`) using the
-      following command:
+   1. Install the protocol compiler plugin for Go using the following commands:
 
       ```sh
       $ export GO111MODULE=on  # Enable module mode
       $ go get github.com/golang/protobuf/protoc-gen-go
+      $ go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.0
```

Comment:
So we still need both of these, right? Well, I can't make it through the (updated) quick start with out both.

Reply:
Correct.  The first binary generates the messages and the second generates the services.

We'll also need (or maybe not "need" but "should"?) to update the first one to refer to `google.golang.org/protobuf` once we convert our generated files in the grpc repo.

Reply:
> ... update the first one to refer to google.golang.org/protobuf once we convert our generated files in the grpc repo.

Will those updates be included in the Oct. 20 release?

Reply:
I don't think there's anything blocking us from doing it, and it's pretty low effort.  I'll try to get it done this week.

Reply:
I've posted a follow up question here: https://github.com/grpc/grpc-go/issues/3928

Reply:
> I don't think there's anything blocking us from doing it, and it's pretty low effort. I'll try to get it done this week.

Actually, I started to do this, and found it's not low effort.  There are several pretty major things that need to be migrated to use the new API, which makes the cost/benefit questionable at this time.  This definitely won't be happening by the next release, and probably not in the next couple quarters.

Reply:
Got it. Thanks for the heads up.